imap/ctl_mboxlist.c

```diff
@@ -1222,7 +1220,30 @@ int main(int argc, char *argv[])
     int dointermediary = 0;
     int undump_legacy = 0;
 
-    while ((opt = getopt(argc, argv, "C:Lawmdurcxf:p:viy")) != EOF) {
+    /* keep this in alphabetical order */
+    static const char *const short_options = "C:Ladf:imp:uvwxy";
```

Comment:
Also no r option here?

Reply:
Also no `c` -- the implementations of these were removed in fc296e2bbf, but I guess I forgot to remove them from the getopt string at the time (seems to be a recurring theme!).  Here, I finally noticed, and finished cleaning it up.

Thanks for noticing, I really appreciate the close eye. :)